notebook/static/notebook/less/cell.less

```diff
@@ -21,6 +21,11 @@ div.cell {
             border-color: transparent;
         }
     }
+    
+    &.marked {
+        border-left-color: blue;
```

Comment:
The blue is a good starting point, but maybe @cameronoelsen has a better blue that we can use?